app/src/main/java/com/zeapo/pwdstore/sshkeygen/ShowSshKeyFragment.kt

```diff
@@ -4,61 +4,36 @@
  */
 package com.zeapo.pwdstore.sshkeygen
 
-import android.annotation.SuppressLint
 import android.app.Dialog
-import android.content.ClipData
+import android.content.Intent
 import android.os.Bundle
-import android.view.View
-import android.widget.TextView
-import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.DialogFragment
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import com.zeapo.pwdstore.R
-import com.zeapo.pwdstore.utils.clipboard
-import java.io.File
+import com.zeapo.pwdstore.git.sshj.SshKey
 
 class ShowSshKeyFragment : DialogFragment() {
 
-    private lateinit var builder: MaterialAlertDialogBuilder
-    private lateinit var publicKey: TextView
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        builder = MaterialAlertDialogBuilder(requireActivity())
-    }
-
-    @SuppressLint("InflateParams")
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
```

Comment:
Can we get rid of this entire fragment? Feels like doing it in the activity itself works just as well.

Reply:
It's shown in two different places though (keygen and preferences) and putting the code next to either would feel out of place. Should we make it a top-level function instead and just show the dialog in the two places?

Reply:
Ah, I completely forgot about the preference case. Let's leave this as-is.